entries/s/swile.co.json

```
@@ -0,0 +1,17 @@
+{
+  "Swile": {
+    "domain": "swile.co",
+    "url": "https://www.swile.co/",
+    "contact": {
+      "facebook": "swileglobal",
+      "twitter": "swile",
+      "language": "fr"
+    },
+    "keywords": [
+      "banking"
+    ],
+    "regions": [
+      "fr"
```

```suggestion
      "fr",
      "br"
```

Swile also operates in Brazil